public/sass/components/_dropdown.scss

```diff
@@ -277,16 +278,21 @@
 .dropdown-submenu.pull-left {
   // Undo the float
   // Yes, this is awkward since .pull-left adds a float, but it sticks to our conventions elsewhere.
-  float: none;
+  float: none !important;
 
   // Positioning the submenu
   > .dropdown-menu {
     left: -100%;
-    margin-left: 10px;
+    width: 100%;
```

Comment:
this was the one that was so hard to figure out, to make the submenu left: -100% positioning work it needs to be as wide as the parent

Reply:
Could using something like `transform: translateX(-100%);` achieve the same result without locking down the width?

Reply:
Tested this now but did not work